pkg/resources/config.go

```diff
@@ -0,0 +1,22 @@
+package resources
+
+import (
+	"os"
+	"strconv"
+	"time"
+)
+
+const DefaulReconcileTime = 300
+
+// returns envar for reconcile time else returns default time
+func GetReconcileTime() time.Duration {
```

Comment:
We should probably have a test for this, it's an important piece of logic in the operator overall, we don't want any regressions or unexpected behaviour here

Reply:
Will catch in a follow up PR